Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM php:8.2-cli
```

Comment:
Lets store this in the same folder as we do for the other projects. How does this get published to packagist again?

Reply:
Moved! 🚚 And I think it automatically publishes when we make a tag in this repo